File: class.cpp
#include<string>

class Event{
private:
  int start,end;
  std::string location;
public:
  Event(int sta,int en,std::string loc);
};

Event::Event(int sta,int en,std::string loc){
    start=sta;end=en;
    this->location=loc;
}

File: class.py
class Event:
    def __init__(self, start, end, loc):
        self.start = start
        self.end = end
        self.location = loc

    def startsWhen(self):
        return self.start

    def endsWhen(self):
        return self.end

    def heldWhere(self):
        return self.location
